vscode/microsoft-kiota/src/types/enums.ts

```diff
@@ -7,34 +7,4 @@ export enum GenerationType {
     ApiManifest = 2,
 };
```

Comment:
why haven't we moved this enum as well?

Reply:
It has not been referenced inside the module

Reply:
right, because we have different JSON RPC methods for that.
If it's not referenced at all, please remove it.

Reply:
It can't be removed as it's used within vscode functions.